scripts/test-dssspell-forge.sh

```diff
@@ -22,12 +23,16 @@ export FOUNDRY_OPTIMIZER=false
 export FOUNDRY_OPTIMIZER_RUNS=200
 export FOUNDRY_ROOT_CHAINID=5
 
-if [[ -z "$MATCH" && -z "$BLOCK" ]]; then
-    forge test --fork-url "$ETH_RPC_URL"
-elif [[ -z "$BLOCK" ]]; then
-    forge test --fork-url "$ETH_RPC_URL" --match-test "$MATCH" -vvv
-elif [[ -z "$MATCH" ]]; then
-    forge test --fork-url "$ETH_RPC_URL" --fork-block-number "$BLOCK"
-else
-    forge test --fork-url "$ETH_RPC_URL" --match-test "$MATCH" --fork-block-number "$BLOCK" -vvv
+EXTRA_ARGS=''
```

Comment:
`FILTERED_ENV_ARGS`, `PERMITTED_ENV_ARGS`, `APPROVED_ENV_ARGS` etc

Reply:
I went with `TEST_ARGS`... it feels weird calling this `ENV_` because they are not environment variables, they are command line arguments.